pkg/skaffold/instrumentation/meter.go

```diff
@@ -66,5 +66,10 @@ func InitMeter(runCtx *runcontext.RunContext, config *latest.SkaffoldConfig) {
 			meter.SyncType[yamltags.GetYamlTag(artifact.Sync)] = true
 		}
 	}
+	meter.Deployers = yamltags.GetYamlTags(config.Deploy.DeployType)
```

Comment:
Are you capturing Dev iterations yet?
If yes,  you also need to add errorCode for current dev iteration
See https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/runner/dev.go#L176 for more information on how Skaffold EventAPI does this.

Reply:
I am not capturing it yet, so that may be more appropriate for the next PR. Capturing errors could probably be included in that PR too instead of this one.